rules/windows/defense_evasion_system_critical_proc_abnormal_file_activity.toml

```diff
@@ -31,11 +31,12 @@ event.category:file and not event.type:deletion and
 framework = "MITRE ATT&CK"
 [[rule.threat.technique]]
 id = "T1133"
-name = "Exploitation for Defense Evasion"
+name = "External Remote Services"
```

Comment:
hmmm

Reply:
Yea this one probably should be converted the other way around -> _to_ 1211 - updating